README.md

```diff
@@ -76,7 +76,10 @@ of AOTriton. The compatibility matrix is shown below
    suffer from runtime errors.
 3. To be specific, it is shipped with 0.9.2b. 0.9b and 0.9.1b should not be
    used in order to avoid linking issues, and confusion about version strings.
-4. 0.10b is backward compatible with 0.9b's API. However, PyTorch 2.8 will
+4. 0.10b is backward compatible with 0.9b's API. Hence it can be used as a drop-in
+   replacement for installed PyTorch wheels. However, 0.10b cannot be built with
```

Comment:
> Hence it can be used as a drop-in replacement for installed PyTorch wheels.

Can you please test this locally on your end? I wouldn't want to make that assertion without trying it at least.